minikube kubectl appends --cluster flag breaking exec commands

Looks like [ #10535](https://github.com/kubernetes/minikube/pull/10535) may have introduced a bug
```
[minikube] ~ $ minikube kubectl -- --context minikube exec ambassador-84ffc9fdc4-nxp4h -- echo "hello"
hello --cluster minikube

[minikube] ~ $ kubectl --context minikube exec ambassador-84ffc9fdc4-nxp4h -- echo "hello"
hello
```

This causes issues when trying to run exec commands because the `--cluster` flag is always appended to the end leading to issues such as

```
[minikube] ~ $ minikube kubectl -- --context minikube exec mysql-db-79b945965d-gtzf7 -- mysqladmin ping

mysqladmin: [ERROR] unknown option '--cluster'
command terminated with exit code 2
``` 
```
[minikube] ~ $ minikube version
minikube version: v1.18.1
commit: 09ee84d530de4a92f00f1c5dbc34cead092b95bc
```

--- a/cmd/minikube/cmd/kubectl.go
+++ b/cmd/minikube/cmd/kubectl.go
@@ -48,7 +48,7 @@ minikube kubectl -- get pods --namespace kube-system`,
 		}
 
 		cluster := []string{"--cluster", ClusterFlagValue()}
-		args = append(args, cluster...)
+		args = append(cluster, args...)
 
 		c, err := KubectlCommand(version, args...)
 		if err != nil {

